src/topic_tree_.toit

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2022 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the LICENSE file.
+
+class TopicTreeNode_:
+  topic_level /string
```

Comment:
This seems like an odd name. I would expect the level to be numeric. Maybe it is just the `segment`?

Reply:
changed to "segment".
done.